code/client/munkiimport

```diff
@@ -760,6 +777,8 @@ def main():
 
     parser = PassThroughOptionParser(usage=usage, epilog=epilog)
 
+    parser.add_option('--init', action='store_true',
+                      help='Initialize a munki repository.')
```

Comment:
OK. It works as I personally would expect - populate an existing directory or create it first. It might indeed be a good idea to return a string saying "New repository created at %s"